pilot/pkg/bootstrap/options.go

```diff
@@ -139,6 +140,7 @@ func NewPilotArgs(initFuncs ...func(*PilotArgs)) *PilotArgs {
 // Apply default value to PilotArgs
 func (p *PilotArgs) applyDefaults() {
 	p.Namespace = PodNamespace
+	p.CniNamespace = p.Namespace
```

Comment:
```suggestion
	p.CniNamespace = PodNamespace
```

ultra-nit: given above comment, just do this.